proj3_chess/player.py

```diff
@@ -0,0 +1,18 @@
+from typing import Callable
+
+
+class Player:
+    def __init__(self, direction: int, symbol: Callable[[str], str]) -> None:
+        """This class represents a player in a chess game. \n
+        direction (int): the direction in which the pieces move - either 1 or -1 \n
+        symbol (function): a string function that recieves a string and returns a string. this will be the graphichal representation for the pieces. example: passig the function upper means the players pieces will be sybmolized by capital letters.
+        """
+        self.direction = direction
+        self.symbol = symbol
+        self.pieces = []
+
+    def all_available_moves(self):
```

Comment:
`get_all_available_moves`. Methods should be named for the action they perform. `all_available_moves` is not an action. If this was a variable that name would make more sense